index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio header & headline section</title>
+</head>
+<body>
+    <header>
+        <a href="#home" class="active">Ahmed Adel</a>
+        <a href="#home" class="icon">
+            <i class="fa fa-bars"></i>
+        </a>        
+
+    </header>
+    <main>
+        <section class="text-content">
+            <h1>Hey There.<br>I'm Ahmed</h1>
+            <h2 class="caption">I’am a Software Developer</h2>
+            <p>I can help you build a product , 
```

Comment:
- [ ] With the live version added in the readme file, this `p` tag is overlaping the sections above. See screenshot below and give it a fix.
![Screenshot from 2021-08-12 15-35-54](https://user-images.githubusercontent.com/65501535/129197753-b6c23248-9906-4c06-80da-6b543a770863.png)

Reply:
at the same directions noting happened while I was testing , Here is screenshot 
![image](https://user-images.githubusercontent.com/43178495/129245947-48661502-d73d-4735-98cc-0e27182bd8da.png)

Reply:
last style
![image](https://user-images.githubusercontent.com/43178495/129285113-cf5a1bcc-69f9-42ed-9060-1683c500732e.png)